mmdet/core/post_processing/bbox_nms.py

```diff
@@ -32,20 +34,32 @@ def multiclass_nms(multi_bboxes,
     if multi_bboxes.shape[1] > 4:
         bboxes = multi_bboxes.view(multi_scores.size(0), -1, 4)
     else:
-        bboxes = multi_bboxes[:, None].expand(-1, num_classes, 4)
+        bboxes = multi_bboxes[:, None].expand(
+            multi_scores.size(0), num_classes, 4)
     scores = multi_scores[:, :-1]
 
     # filter out boxes with low scores
     valid_mask = scores > score_thr
-    bboxes = bboxes[valid_mask]
+
+    # (TODO): as ONNX does not support repeat now,
```

Comment:
We may also state it's equivalent workaround when not exploring ONNX in comments.

Reply:
We may add something link
```python
# We use masked_select for ONNX exporting purpose, which is equivalent to
bboxes = bboxes[valid_mask]
```